### ISSUE TITLE
Reg. Wizard: Upon clicking proceed the new stage should appear from the very top

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-deals-dapp#1092]
Steps to reproduce:
1. Navigate to open proposal or partnered deal reg.wiz: https://prime-deals-dapp-git-refac-deals-au2-curvelabs.vercel.app/initiate/token-swap/open-proposal/lead-details
2. fill in details and click proceed.

Currently:
The new stages appear from the middle
[Uploading... Create-an-Open-Proposal-Lead-Details-Prime-Deals.png]

Expected:
New stage should appear from the top

**Comment by BartuCulha** [comment]
I am able to reproduce here:
https://user-images.githubusercontent.com/24368665/182591288-e4c78b9c-90b2-4d33-a4ca-1d671adee00a.mp4

**Comment by giladt** [comment]
Well, it's not a BUG, it's a feature (BTW this behavior is taken from Launch)!

### Explanation:
The app keeps the last position of a visited page, so if you navigate back to this page, it will go to the same position you've left it.
On your first visit to the app- you'll see the top of the next stage when pressing proceed, but if you've visited this stage before, it will scroll to the last position you've left it (positions of each page are stored in the local-storage).
@BartuCulha @blink-km @hiaux0

**Comment by BartuCulha** [comment]
yeah i thought we ditched that idea because it's not a great UX :D

**Comment by giladt** [comment]
Moving to `Accepted` as current behavior mimics AU1.
### Other options to consider:
* Revert- Means of removing the part that stores/retrieve the scroll position of a page in the local storage, and letting the browser handle it natively. Note that for sub-routes it means page position after navigation is the same as on the previous page (original issue)
* Refactor as initially asked- Revert and add scroll to top after pressing Proceed (Needs investigation of complexity)

cc: @hiaux0
